mypy/subtypes.py

```diff
@@ -143,6 +143,8 @@ def visit_tuple_type(self, left: TupleType) -> bool:
                 return all(is_subtype(item, target_item_type)
                            for item in left.items)
             elif (is_named_instance(right, 'typing.Iterable') or
+                  is_named_instance(right, 'typing.Sized') or
```

Comment:
Try the 3rd commit.

On Mon, Dec 14, 2015 at 3:06 PM, Guido van Rossum <guido@python.org> wrote:

> Hm... Weird. If I make Sized non-Generic I indeed get an internal error.
> But if I remove the check for typing.Sized from subtypes.py, the test I
> wrote fails, with
> ```
> error: Incompatible types in assignment (expression has type "Tuple[]",
> variable has type "Sized")
> ```
> I think I have a fix, wait a sec.
>



-- 
--Guido van Rossum (python.org/~guido)